mlflow/projects/__init__.py

```diff
@@ -283,11 +267,15 @@ def _maybe_create_conda_env(conda_env_path):
     (_, stdout, _) = process.exec_cmd([conda_path, "env", "list", "--json"])
     env_names = [os.path.basename(env) for env in json.loads(stdout)['envs']]
 
-    conda_action = 'create'
-    if conda_env not in env_names:
-        eprint('=== Creating conda environment %s ===' % conda_env)
-        process.exec_cmd([conda_path, "env", conda_action, "-n", conda_env, "--file",
-                          conda_env_path], stream_output=True)
+    project = _project_spec.load_project(project_dir)
```

Comment:
It seems like this method should take a project, instead of project_dir, since its caller has one.

Reply:
Oops my mistake :P